setup.py

```diff
@@ -41,8 +41,8 @@
         "console_scripts": [
             "o2r.new=Ot2Rec.main:new_proj",
 
-            "o2r.mc.new=Ot2Rec.main:create_mc2_yaml",
-            "o2r.mc.run=Ot2Rec.main:run_mc2",
+            "o2r.mc.new=Ot2Rec.main:mc2_create_yaml",
```

Comment:
could we use Ot2Rec.motioncorr:create_yaml as the entry point? Then we might not even need to wrap it in main

Reply:
That's a brilliant idea I haven't thought about!